### PR TITLE
WSSQL-152 Execute prepared processes quoted input params

### PR DESCRIPTION
**Pull request by rpastrana**
- Strips quotes from input params

Signed-off-by: Rodrigo Pastrana <rodrigo.pastrana@lexisnexis.com>

### CODE REVIEW

**Comment by rpastrana** [comment]
@RussWhitehead please review

File: esp/services/ws_sql/ws_sqlService.cpp [hunk 1]
```diff
@@ -1179,10 +1179,31 @@ void CwssqlEx::createWUXMLParams(StringBuffer & xmlparams, HPCCSQLTreeWalker* pa
...
+
+                if (value && *value)
+                {
+                    while(value && isspace(*value)) //fast trim left
+                        value++;
+
+                    int len = strlen(value);
+
+                    while(len && isspace(value[len-1])) //fast trim right
+                        value[--len] = '\0';
```
Comment (RussWhitehead) [inline esp/services/ws_sql/ws_sqlService.cpp]: Consider remembering the offset of most recently found (leftmost) space, and after you exit the loop set it to NULL

File: esp/services/ws_sql/ws_sqlService.cpp [hunk 2]
```diff
@@ -1179,10 +1179,31 @@ void CwssqlEx::createWUXMLParams(StringBuffer & xmlparams, HPCCSQLTreeWalker* pa
...
+
+                    //WU cloning mechanism doesn't handle quoted strings very well...
+                    //We're forced to blindly remove them here...
+                    if (value[0] == '\'' && value[len-1] == '\'')
+                    {
+                        value[len-1] = '\0'; //clip rightmost quote
+                        value++; //clip leftmost quote
```
Comment (RussWhitehead) [inline esp/services/ws_sql/ws_sqlService.cpp]: If token is a single quote (len == 1) this won't be right

Comment (RussWhitehead) [inline esp/services/ws_sql/ws_sqlService.cpp]: Also problematic if token is 2 quote marks  (len == 2)

**Comment by rpastrana** [comment]
@RussWhitehead please review.

**Comment by RussWhitehead** [comment]
Looks good

**Pull request merged**